docs/nf4_science/imaging/01_basics.md

```diff
@@ -0,0 +1,413 @@
+
+executor >  local (1)
+[a3/7be2fa] sayHello | 1 of 1 ✔
+```
+
+Congratulations, you just ran your first Nextflow workflow!
+
+The most important output here is the last line (line 6):
+
+```console title="Output" linenums="6"
+[a3/7be2fa] sayHello | 1 of 1 ✔
+```
+
+This tells us that the `sayHello` process was successfully executed once (`1 of 1 ✔`).
+
+That's great, but you may be wondering: where is the output?
+
+### 1.2. Find the output file in the `results` directory
+
+This workflow is configured to publish its output to a directory called `results`.
+If you look at your current directory, you will see that when you ran the workflow, Nextflow created a new directory called `results`, which contains a file called `output.txt`.
+
+```console title="results/" linenums="1"
+results
+└── output.txt
+```
+
+Open the file; the contents should match the string you specified on the command line.
+
+<details>
+  <summary>File contents</summary>
+
+```console title="results/output.txt" linenums="1"
+Hello World!
+```
+
+</details>
+
+That's great, our workflow did what it was supposed to do!
+
+However, be aware that the 'published' result is a copy (or in some cases a symlink) of the actual output produced by Nextflow when it executed the workflow.
+
+So now, we are going to peek under the hood to see where Nextflow actually executed the work.
+
+!!! Warning
+
+    Not all workflows will be set up to publish outputs to a results directory, and/or the directory name may be different.
+    A little further in this section, we will show you how to find out where this behavior is specified.
+
+### 1.3. Find the original output and logs in the `work/` directory
+
+When you run a workflow, Nextflow creates a distinct 'task directory' for every single invocation of each process in the workflow (=every step in the pipeline).
+For each one, it will stage the necessary inputs, execute the relevant instruction(s) and write outputs and log files within that one directory, which is named automatically using a hash in order to make it unique.
+
+All of these task directories will live under a directory called `work` within your current directory (where you're running the command).
+
+That may sound confusing, so let's see what that looks like in practice.
+
+Going back to the console output for the workflow we ran earlier, we had this line:
+
+```console title="Excerpt of command output" linenums="6"
+[a3/7be2fa] sayHello | 1 of 1 ✔
+```
+
+See how the line starts with `[a3/7be2fa]`?
+That is a truncated form of the task directory path for that one process call, and tells you where to find the output of the `sayHello` process call within the `work/` directory path.
+
+You can find the full path by typing the following command (replacing `a3/7be2fa` with what you see in your own terminal) and pressing the tab key to autocomplete the path or adding an asterisk:
+
+```bash
+ls work/a3/7be2fa*
```

Comment:
```suggestion
tree -a work/a3/7be2fa*
```
you switch a lot between 'ls' and 'tree' - this tutorial might contain a lot of non-tech savvy, folks, i would say just pick one or the other or you may end up losing folks out of confusion